test/test_poolmanager.py

```diff
@@ -366,3 +373,42 @@ def test_pool_manager_no_url_absolute_form(self):
         p = PoolManager()
         assert p._proxy_requires_url_absolute_form("http://example.com") is False
         assert p._proxy_requires_url_absolute_form("https://example.com") is False
+
+    @pytest.mark.skipif(sys.version_info < (3, 7), reason="requires Python 3.7+")
+    def test_normalizer_defaults_blocksize_to_16384(self):
+        """Assert normalizer defaults to blocksize 16384"""
+        p = PoolManager()
+
+        pool_blocksize_16384 = p.connection_from_url(
+            "http://example.com", {"blocksize": 16384}
+        )
+        assert pool_blocksize_16384.conn_kw["blocksize"] == 16384
+
+        pool_blocksize_not_set = p.connection_from_url("http://example.com")
+        assert pool_blocksize_not_set == pool_blocksize_16384
+
+        pool_blocksize_set_as_none = p.connection_from_url(
+            "http://example.com", {"blocksize": None}
```

Comment:
It is interesting what should happen here. It could be argued that explicit `None` should not be treated the same as an omission (->setting default), but I'm not sure what's better design.